src/antares/tools/matrix_tool.py

```diff
@@ -36,3 +40,17 @@ def df_save(df: pd.DataFrame, path: Path) -> None:
 
 def df_read(path: Path) -> pd.DataFrame:
     return pd.read_csv(path, sep="\t", header=None)
+
+
+def read_timeseries(ts_file_type: TimeSeriesFileType, study_path: Path, area_id: Optional[str] = None, constraint_id: Optional[str] = None, renewable_id: Optional[str] = None,) -> pd.DataFrame:
```

Comment:
As st_storage, thermal and renewable are all clusters, it could be nice to rename this parameter `cluster_id` otherwise you'll have to add other arguments here. This goes with changing `RENEWABLE_DATA_SERIES` to put `cluster_id` inside it. 

After this it will be good for me

Reply:
No problem, i changed mentions of renewable_id for cluster_id, for more clarity